src/unix/aix-common.c

```diff
@@ -170,6 +172,10 @@ int uv_interface_addresses(uv_interface_address_t** addresses, int* count) {
     return UV__ERR(errno);
   }
 
+  if (0 > (sock6fd = socket(AF_INET6, SOCK_DGRAM, IPPROTO_IP))) {
+    return UV__ERR(errno);
+  }
+
   if (ioctl(sockfd, SIOCGSIZIFCONF, &size) == -1) {
     uv__close(sockfd);
```

Comment:
Should `sock6fd` be closed here and around line 187?

Reply:
Should be fixed now (both cases).